CRVResponse/fcl/prolog_v10.fcl

```diff
@@ -146,10 +147,12 @@ BEGIN_PROLOG
                                         //e.g. 0.985 for a leading edge at 50% pulse height
                                         //e.g. 1.385 for a leading edge at 20% pulse height
                                         //e.g. 1.587 for a leading edge at 10% pulse height
-      allowDoubleGumbel         : false //tries fitting with two Gumbel functions (needs a very small "minPulseHeightRatio", e.g. 0.01)
-      doubleGumbelThreshold     : 2.0   //Chi2/#ADCsamples (based on a single Gumble fit)
-                                        //at which a fit with two Gumbel functions should be attempted
-      minPEs                    : 0     //0 PEs
+      pulseThreshold            : 0.5   //50% of ADC peak value above pedestal
+                                        //threshold used to determine the pulse time interval for the no-fit option
+      pulseAreaThreshold        : 5     //5 ADC above pedestal
```

Comment:
Is it really an area threshold as the name suggests?
We can't zero suppress the data on the charge.

Reply:
This threshold determines the range where the ADC values of a pulse are summed up. If the waveform crosses this threshold at the beginning of the pulse, we start adding the ADC counts to calculate the PE value (in the no-fit option) until the waveform falls below this threshold at the end of the pulse. I should change the explanation in the fcl file.

Reply:
Rob, are you suggesting he remove these comments?  I personally find a description of the meaning of the variable helpful.  Maybe you mean he should remove the value from the comment?

Reply:
Perhaps I misundertood but I understood the request to be for something that, if done properly, would be many lines long.  If a meaningful short comment will do the job, then I agree that having it inline is preferred.  I was imagining a case in which a meaningful comment would be many lines long.

Reply:
This is just a suggestion, not a requirement.  So I will approve.